src/Sentry.Maui.CommunityToolkitMvvm/CtMvvmMauiElementEventBinder.cs

```diff
@@ -0,0 +1,151 @@
+                break;
+
+            case SearchBar searchBar:
+                TryBindTo(searchBar.SearchCommand, bind);
+                break;
+
+            default:
+                TryGestureBinding(element, bind);
+                break;
+        }
+    }
+
+    private static void TryGestureBinding(VisualElement element, bool bind)
+    {
+        if (element is IGestureRecognizers gestureRecognizers)
+        {
+            foreach (var gestureRecognizer in gestureRecognizers.GestureRecognizers)
+            {
+                TryBindTo(gestureRecognizer, bind);
+            }
+        }
+    }
+
+    private static void TryBindTo(IGestureRecognizer recognizer, bool bind)
+    {
+        switch (recognizer)
+        {
+            case TapGestureRecognizer tap:
+                TryBindTo(tap.Command, bind);
+                break;
+
+            case SwipeGestureRecognizer swipe:
+                TryBindTo(swipe.Command, bind);
+                break;
+
+            // no commands
+            //case PinchGestureRecognizer pinch
+            //case PanGestureRecognizer pan
+            case DragGestureRecognizer drag:
+                TryBindTo(drag.DragStartingCommand, bind); // unlikely to ever be async
+                TryBindTo(drag.DropCompletedCommand, bind);
+                break;
+
+            case PointerGestureRecognizer pointer:
+                TryBindTo(pointer.PointerPressedCommand, bind);
+                TryBindTo(pointer.PointerReleasedCommand, bind);
+                TryBindTo(pointer.PointerEnteredCommand, bind); // unlikely to ever be async
+                TryBindTo(pointer.PointerExitedCommand, bind);
+                break;
+        }
+    }
+
+    private static List<IAsyncRelayCommand> _refs = [];
+    private static void TryBindTo(ICommand? command, bool bind)
+    {
+        if (command is IAsyncRelayCommand relayCommand)
+        {
+            if (bind)
+            {
+                // necessary for collectionview buttons
+                if (!_refs.Contains(relayCommand))
+                {
+                    _refs.Add(relayCommand);
+                    relayCommand.PropertyChanged += RelayCommandOnPropertyChanged;
+                }
+            }
+            else
+            {
+                _refs.Remove(relayCommand);
+                relayCommand.PropertyChanged -= RelayCommandOnPropertyChanged;
+            }
+        }
+    }
+
+    private static ConcurrentDictionary<IAsyncRelayCommand, (ITransactionTracer Transaction, ISpan Span)> _contexts = new();
+
+    private static void RelayCommandOnPropertyChanged(object? sender, PropertyChangedEventArgs e)
+    {
+        if (e.PropertyName == nameof(IAsyncRelayCommand.IsRunning))
+        {
+            var relay = (IAsyncRelayCommand)sender!;
+
+            if (relay.IsRunning)
+            {
+                var transaction = SentrySdk.StartTransaction("ctmvvm", "asynccommand");
+                var span = transaction.StartChild("run");
```

Comment:
Since the span and the transaction have the same start and end time, it seems like we could get away without the span.

A transaction _is like a span_ and we'll measure timing the same way at the transaction level

Reply:
No problems.  You're the dude that has been telling me to use spans "cause transactions are going away".  I can whatever here :)

Reply:
Right, use spans if you expect those operations to be attached to a transaction at some point. E.g: An HTTP integration, attaches spans to a transaction when the transaction happens. Eventually we'll send them stand alone, when we add support for that. But today they require a transaction.

But if you're starting the transaction somewhere, you have a 'stopwatch' there already. You can have child spans for it, but if they have the same start/end time as the transaction, you can skip that child span, the transction itself works as a stop watch.

Reply:
If/when we get rid of transactions, we've going to have to go through all our automatic instrumentation and replace any transactions that we have with parentless spans... until then though, the only way to create a span without a parent and send this to Sentry is to create a transaction.